src/json-completion.ts

```diff
@@ -38,8 +48,15 @@ class CompletionCollector {
   }
 }
 
+type JSONCompletionOptions = {
+  mode?: "json" | "json5";
```

Comment:
unfortunately, `CompletionContext` nor `EditorState` give us metadata about which parser has parsed the editor state, so we must rely on users to provide this information